.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "module/OpenSTA"]
-	path = src/sta
-	url = ../../The-OpenROAD-Project/OpenSTA.git
 [submodule "src/abc"]
 	path = third-party/abc
 	url = ../../The-OpenROAD-Project/abc.git
+[submodule "src/sta"]
+	path = src/sta
+	url = ../../The-OpenROAD-Project/OpenSTA.git
```

Comment:
The change to the .gitmodules is back.  Please undo this.

Reply:
Done